buildkite/src/Constants/ContainerImages.dhall

```diff
@@ -5,8 +5,6 @@
 { toolchainBase = "codaprotocol/ci-toolchain-base:v3"
 , minaToolchainBullseye =
     "gcr.io/o1labs-192920/mina-toolchain@sha256:8248ceb8f35bae0b5b0474a3e296ad0380ea2ab339f353943ee36564a12a745a"
-, minaToolchainBookworm =
-    "gcr.io/o1labs-192920/mina-toolchain@sha256:8248ceb8f35bae0b5b0474a3e296ad0380ea2ab339f353943ee36564a12a745a"
 , minaToolchain =
     "gcr.io/o1labs-192920/mina-toolchain@sha256:8248ceb8f35bae0b5b0474a3e296ad0380ea2ab339f353943ee36564a12a745a"
 , elixirToolchain = "elixir:1.10-alpine"
```

Comment:
What is with elixirToolchain on alpine?

Reply:
Looks like not used. I will remove it in next PR